web/webpack.config.js

```diff
@@ -46,7 +46,9 @@ const copy_files = [
 const plugins = [
   new Copy({ patterns: copy_files }),
   new Extract({ filename: "[name].css" }),
-  new CockpitPoPlugin(),
+  // the wrapper sets the main code called in the po.js files,
+  // the PO_DATA tag is replaced by the real translation data
+  new CockpitPoPlugin({ wrapper: "agama.locale(PO_DATA);" }),
```

Comment:
Note: The default wrapper is `agama.locale(PO_DATA)` which calls the Cockpit implementation.